addScript.js

```diff
@@ -4,14 +4,21 @@
 */
 module.exports = function(src) {
 	try {
-		if (typeof eval !== "undefined") {
-			eval.call(null, src);
-		} else if (typeof execScript !== "undefined") {
+		if (typeof execScript !== "undefined"
+			/* "bad" IE */ && typeof attachEvent !== "undefined" && typeof addEventListener === "undefined") {
```

Comment:
Please extract the IE Check into its own block/helper

```js
// Check for IE =< 8
function isIE () {
  return typeof ...
}
```

Reply:
Move the helper above L5

Reply:
It's done